### PR TITLE
Added support to skopeo-copy Task for copying multiple images.

### PR DESCRIPTION
**Pull request by Senjuti256**
Added support to skopeo-copy Task for copying multiple images. This is done by using images_url workspace. 
Signed off by @Senjuti256 <sde@redhat.com>

### CODE REVIEW

File: scripts/skopeo-common.sh [hunk 1]
```diff
@@ -16,7 +20,8 @@ exported_or_fail \
     PARAMS_SOURCE \
     PARAMS_DESTINATION \
     RESULTS_SOURCE_DIGEST_PATH \
-    RESULTS_DESTINATION_DIGEST_PATH
+    RESULTS_DESTINATION_DIGEST_PATH \
```
Comment (savitaashture) [inline scripts/skopeo-common.sh:23]: does `\` required at the end ??

Comment (Senjuti256) [reply scripts/skopeo-common.sh]: Corrected it @savitaashture

**Pull request merged**